R/utils.R

```diff
@@ -2,6 +2,17 @@ last <- function(x) x[length(x)]
 
 sq <- function(x) glue::single_quote(x)
 
+sglue <- function(...) stop(glue::glue(..., .envir = parent.frame()), call. = FALSE)
```

Comment:
Are you doing `glue(..., .envir = parent.frame())` instead of `glue(...)` for a specific reason? I would have expected the latter.